site/content/docs/0.0/customize/custom-libraries.md

```diff
@@ -12,19 +12,21 @@ toc: true
 **This content is not intended for standalone projects.** It is specifically for those looking to customize the entire library to fit a completely different design or brand.
 {{< /callout >}}
 
-## Architecture
+## Tokens architecture
+
+OUDS Web is built on the basis of design tokens that capture all the design decisions utilized within our design system. They ensure the consistency of components between them and facilitate the configuration of the library.
 
 ### Raw tokens
 
-The [`scss/tokens/_raw.scss`]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/scss/tokens/_raw.scss) file consolidates all basic tokens. While these tokens don't carry semantic meaning and aren't meant to be used directly, they form the foundation of the Orange Unified Design System. Their primary purpose is to be utilized by semantic tokens, and occasionally, by component tokens.
+The [`scss/tokens/_raw.scss`]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/scss/tokens/_raw.scss) file consolidates all basic tokens. While these tokens don't carry semantic meaning and aren't meant to be used directly in projects, they form the foundation of the Orange Unified Design System. Their primary purpose is to be utilized by semantic tokens, and occasionally, by component tokens.
 
 ### Semantic tokens
 
-The [`scss/tokens/_semantic.scss`]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/scss/tokens/_semantic.scss) file contains all the semantic tokens. Unlike raw tokens, these are intended to be used directly. They are built on raw tokens, assigning them semantic meaning through media queries or specific contexts. These tokens are ready to be applied either in utilities or directly within component tokens.
+The [`scss/tokens/_semantic.scss`]({{< param repo >}}/blob/v{{< param current_version >}}-ouds-web/scss/tokens/_semantic.scss) file contains all the semantic tokens. Unlike raw tokens, these are intended to be used directly in projects. They are built on raw tokens, assigning them semantic meaning through media queries or specific contexts. These tokens are ready to be applied either in utilities or directly within component tokens. The names of these tokens are not intended to be modified for a custom library, only the correspondence between a token and its value, which comes from the raw tokens.
```

Comment:
> They are built on raw tokens, assigning them semantic meaning through media queries or specific contexts. These tokens are ready to be applied either in utilities or directly within component tokens.

I'm not really in favor of this change (adding "tokens"). They can be used in components without being in a component token. It's not 100% of the time a component token that will use them.


> only the correspondence between a token and its value, which comes from the raw tokens.

Proposal: "only the link between a token and its value defined by a raw token."

Reply:
1. I didn't add "tokens", it was already there. 
2. ok

Reply:
> I didn't add "tokens", it was already there.

Oh, you're right, sorry, badly read the diff 😬 Anyway, I think it's not accurate.

Reply:
Better now?